tests/lib.rs

```diff
@@ -58,7 +58,8 @@ fn onesignal_transaction<D: Deliverable>(deliverable: D) -> Transaction<D> {
 fn httpbin_transaction<D: Deliverable>(deliverable: D) -> Transaction<D> {
     Transaction::new(
         deliverable,
-        Request::get("http://httpbin:80/ip")
+        // This needs to be localhost if run locally
+        Request::get("http://httpbin:8000/ip")
```

Comment:
I see that your `script/test` runs tests locally, does it mean it wont't work?

Reply:
That only works if run _inside the code container_. You can run some tests outside but other fail.